src/test/java/com/google/api/generator/engine/ast/ArithmeticOperationExprTest.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+import static org.junit.Assert.assertThrows;
+
+import org.junit.Test;
+
+public class ArithmeticOperationExprTest {
+  @Test
+  public void concatOperator_validBasic() {
+    VariableExpr lhsExpr =
```

Comment:
Could you please confirm if `x + getSomeString()` is valid?

Reply:
added.

Reply:
I am confused what need to confirm for `x + getSomeString()` here. It is valid in Java, 
```
  private static String getString() {
    return "ab";
  }
int x = 5;
String c = x + getString();
```
I have added a writer test [here](https://github.com/googleapis/gapic-generator-java/pull/260/files#diff-9dd064308f18929ebfe29d140d59466eR369)

Reply:
Thanks for checking.